docs/concepts/namespaces.md

```diff
@@ -4,13 +4,15 @@ id: namespaces
 
 # Namespaces
 
-Namespaces provide users with the exclusive capability to publish contracts under their designated namespaces, similar to GitHub's user and organization model.
+Namespaces provide users with the exclusive capability to publish contracts under their designated namespaces,
+similar to GitHub's user and organization model.
 
-This feature is currently a work in progress (WIP). To learn more about namespaces, please checkout https://github.com/gnolang/gno/issues/1107.
+:::warn This feature isn't enabled by default on the chain and is currently only on test4.gno.land.
```

Comment:
This will not render properly:
```suggestion
:::warn Not enabled

This feature isn't enabled by default on the chain and is currently only on test4.gno.land.

:::
```